pkg/datarepair/irreparabledb/test/irreparabledb_test.go

```diff
@@ -34,30 +35,34 @@ func TestPostgres(t *testing.T) {
 	defer ctx.Cleanup()
 
 	// creating in-memory db and opening connection
```

Comment:
These tests belong to the irreparable_db implementation not the interface. Move tests into `satellite/irreparable_test.go`

Reply:
Although the interface expected behaviour does belong here. Hmm... so, either is fine for now

Reply:
renamed the file to irreparable_test.go